nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/templates/nf-templates.js

```diff
@@ -95,9 +95,13 @@
                     if (clusterSummary.connectedToCluster) {
                         deferred.resolve(false);
                     } else {
-                        nfDialog.showDisconnectedFromClusterMessage(function () {
+                        if (clusterSummary.clustered) {
+                            nfDialog.showDisconnectedFromClusterMessage(function () {
+                                deferred.resolve(true);
+                            });
+                        } else {
                             deferred.resolve(true);
-                        });
+                        }
```

Comment:
The boolean that is resolved above should indicate if the user acknowledged the node disconnection. In the else block, I think we should resolve with false simply because we never prompted the user. From a functional standpoint, it may not actually matter I don't think the flag is inspected when the instance is not clustered. That said, it should try to keep the flag straight on the client side. In the usage on the canvas, the value defaults to false and is only ever updated once the connection state has changed.

This comment applies to the other change here as well.

Reply:
ok, i'll get it fixed.